### DISCUSSION TITLE
The position of Task Manager/Properties in the Task bar context menu.

### DISCUSSION THREAD

**Discussion by HubKing** [valinet/ExplorerPatcher#848] [General]
I have been keep opening the "Properties" when I intended to open Task Manager, because of their positions. On Windows 10, it is at the third position from the bottom, but with this utility, "Properties" is at the third position and "Task Manager" is at the fourth. I have been clicking the third position for so many years and since I also use other Windows 10 computers, I just keep clicking the third position. Do other people experience this? If so...

Do people often change Taskbar settings? In my case, after I change it initially on Windows installation, I rarely change it. So, how about "Taskbar settings" opens current "Properties" instead,, and the "Properties" dialogue has a link like "Open Windows 11's Taskbar settings" the bottom?

Or it adding an option to change the "Properties" position somewhere else, like the top of the menu, or to remove the "Properties" from the context menu and invoke the "Properties" with some other method?

<img src="https://www.howtogeek.com/wp-content/uploads/2019/02/VirtualBox_Windows-10-Pro-1803_01_02_2019_01_35_21.png"/>
<img src="https://i0.wp.com/gearupwindows.com/wp-content/uploads/2022/01/Classic-Taskbar-in-Windows-11.png"/>

**Comment by valinet** [comment]
The option to remove the item is already a program option.

Changing its location or repurposing other items NOW is too disruptive. People that use EP got used to “Taskbar settings” opening W11 Settings and “Properties” opening EP settings. What you suggest is fine, but trust me, it will confuse a lot of folks.

So yeah, as I said, someone else mentioned the same muscle memory thing in the past, I understood perfectly, implemented the option to remove the item from the menu, which I think gets around this problem more than fine. There is still the option to add it to W10 Win-X menu, which might be less disturbing, or just open it manually (rundll32 C:\Windows\dxgi.dll,ZZGUI). The default is like this because it is easy for new users who might not know those tricks, and because the taskbar menu I have access to alter it for both W10 and W11 taskbars, so they always find that there no matter what they use.

I thought about having it at the top but it would feel too disconnected with the rest of the menu, plus, logically, its current position makes sense. There are a ton of exotic ideas that came to my mind, for example, I’d get rid of the window manipulation shortcuts, and then it could stay at the top just fine, but I am sure there still is someone somewhere out there who still uses those.

Thanks

**Comment by HubKing** [comment]
I found the option to hide it in the Advanced section, but it was difficult to find the option to add it to the Start context menu, because the phrase was "Program Settings" there. I think it probably is a good idea somehow to let the user know the other ways to access "Properties" when he removes it in the Advanced section, otherwise he might not be able to find a way to access it himself after closing Properties.

I think I will use the `rundll32 C:\Windows\dxgi.dll,ZZGUI` method, because I happen also to use the Start context menu's "shutdown or sigh out" often, and adding "Properties" there changes its position. I will create a shortcut with that command myself, but how about adding a shortcut in the Start menu (not in the context menu, but just like other installed applications, so that the user could search by the application's name) when installing Explorer Patcher? Almost all applications add shortcuts to the Start menu when they are installed, so it would not be an obtrusive behaviour.

Thanks for your support.

**Reply by valinet** [reply]
Yeah, I'll think about it. Thanks.

**Reply by valinet** [reply]
Implemented a Start menu shortcut in the latest pre-release (42.11).
